plugins/modules/jenkins_plugin.py

```diff
@@ -195,6 +195,29 @@
     url_password: p4ssw0rd
     url: http://localhost:8888
 
+#
+# Example of how to authenticate with serverless deployment
+#
+- name: Update plugins on ECS Fargate Jenkins instance
+  community.general.jenkins_plugin:
+    # plugin name and version
+    name: ws-cleanup
+    version: 0.45
```

Comment:
Versions that look like floats should better be quoted. While here it shouldn't make a difference, this can cause a lot of problems (like `0.10` ending up as `0.1`).
```suggestion
    version: "0.45"
```

Reply:
Thanks Felix, agreed and corrected